go/vt/vtgate/evalengine/fn_time.go

```diff
@@ -1249,6 +1254,41 @@ func (call *builtinLastDay) compile(c *compiler) (ctype, error) {
 	return ctype{Type: sqltypes.Date, Flag: arg.Flag | flagNullable}, nil
 }
 
+func (b *builtinToDays) eval(env *ExpressionEnv) (eval, error) {
+	date, err := b.arg1(env)
+	if err != nil {
+		return nil, err
+	}
+	if date == nil {
+		return nil, nil
+	}
+	dt := evalToDateTime(date, -1, env.now, env.sqlmode.AllowZeroDate())
```

Comment:
We also have `evalToDate`, should that be used here instead? The compiler uses `Convert_xD` which convers to `Date` as well (vs `Convert_xDT` which is to `DateTime`). We should be consistent between the compiler and evaluator.

Reply:
true, used `evalToDate` instead of `evalToDateTime`